lib/tasks/email_missed_reservations.rake

```diff
@@ -2,18 +2,23 @@ desc 'Send emails about missed reservations'
 task email_missed_reservations: :environment do
   # get all reservations that are approved and missed, send an email to
   # user to inform them
-  if AppConfig.first.send_notifications_for_deleted_missed_reservations
-    missed_reservations = Reservation.missed.not_flagged(:missed_email_sent)
+  email_missed_reservations
+end
+
+def email_missed_reservations
+  return unless AppConfig.first
```

Comment:
Similar comment here (as in the check-out reminder task) - let's use `if ... else` blocks and log a message stating that each setting is turned off. I think it would be useful if debugging these tasks in the future.